dropwizard-logging/src/main/java/io/dropwizard/logging/FileAppenderFactory.java

```diff
@@ -95,6 +106,8 @@
     @Min(1)
     private int archivedFileCount = 5;
 
+    private String maxFileSize;
```

Comment:
Could we use Dropwizard's Size class here instead of a string? Then we can enforce validation.